virtcontainers/qemu_ppc64le.go

```diff
@@ -26,7 +26,7 @@ const defaultQemuPath = "/usr/bin/qemu-system-ppc64le"
 
 const defaultQemuMachineType = QemuPseries
 
-const defaultQemuMachineOptions = "accel=kvm,usb=off"
+const defaultQemuMachineOptions = "accel=kvm,usb=off,cap-cfpc=broken,cap-sbbc=broken,cap-ibs=broken"
```

Comment:
Can you confirm that these new options will still work with the older version of qemu?

Reply:
I checked locally and it seems to be working.
```
root     49527 23.4  0.5 2670976 355584 ?      SLl  15:43   0:05 /usr/bin/qemu-system-ppc64 -name sandbox-6aa89a1b2225848c2254b6241f1b304dd88865f6bb65cd14d278f252e0c82e55 -uuid 1141e5b2-1ab8-41a3-8cb9-60a77816bc7e -machine pseries,accel=kvm,usb=off,cap-cfpc=broken,cap-sbbc=broken,cap-ibs=broken -cpu host -qmp unix:/run/vc/vm/6aa89a1b2225848c2254b6241f1b304dd88865f6bb65cd14d278f252e0c82e55/qmp.sock,server,nowait -m 2048M,slots=10,maxmem=33280M -device pci-bridge,bus=pci.0,id=pci-bridge-0,chassis_nr=1,shpc=on,addr=2,romfile= -device virtio-serial-pci,disable-modern=false,id=serial0,romfile= -device virtconsole,chardev=charconsole0,id=console0 -chardev socket,id=charconsole0,path=/run/vc/vm/6aa89a1b2225848c2254b6241f1b304dd88865f6bb65cd14d278f252e0c82e55/console.sock,server,nowait -device virtio-scsi-pci,id=scsi0,disable-modern=false,romfile= -object rng-random,id=rng0,filename=/dev/urandom -device virtio-rng,rng=rng0,romfile= -device virtserialport,chardev=charch0,id=channel0,name=agent.channel.0 -chardev socket,id=charch0,path=/run/vc/vm/6aa89a1b2225848c2254b6241f1b304dd88865f6bb65cd14d278f252e0c82e55/kata.sock,server,nowait -device virtio-9p-pci,disable-modern=false,fsdev=extra-9p-kataShared,mount_tag=kataShared,romfile= -fsdev local,id=extra-9p-kataShared,path=/run/kata-containers/shared/sandboxes/6aa89a1b2225848c2254b6241f1b304dd88865f6bb65cd14d278f252e0c82e55,security_model=none -netdev tap,id=network-0,vhost=on,vhostfds=3,fds=4 -device driver=virtio-net-pci,netdev=network-0,mac=02:42:ac:11:00:02,disable-modern=false,mq=on,vectors=4,romfile= -global kvm-pit.lost_tick_policy=discard -vga none -no-user-config -nodefaults -nographic -daemonize -kernel /usr/share/kata-containers/vmlinux.container -initrd /usr/share/kata-containers/kata-containers-initrd.img -append tsc=reliable no_timer_check rcupdate.rcu_expedited=1 noreplace-smp reboot=k console=hvc0 console=hvc1 cryptomgr.notests net.ifnames=0 quiet panic=1 nr_cpus=10 agent.use_vsock=false -pidfile /run/vc/vm/6aa89a1b2225848c2254b6241f1b304dd88865f6bb65cd14d278f252e0c82e55/pid -smp 1,cores=1,threads=1,sockets=10,maxcpus=10
```
```
root@kata1:/home/nitesh/split_patch/src/github.com/kata-containers/runtime#  /usr/bin/qemu-system-ppc64 --version
QEMU emulator version 2.11.2 (v2.11.2-dirty)
Copyright (c) 2003-2017 Fabrice Bellard and the QEMU Project developers
```